languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/style.xml

```diff
@@ -10188,6 +10188,30 @@ USA
         </rulegroup>
 
 
+        <rule id='CIENTÍFICO_DAR_CONFERIR' name="[Universitário][Científico] dar → conferir" type="style" tags="picky" tone_tags="academic" default='temp_off'>
+            <!--
+            It uses filters:
+              -de → velocidade, integridade, etc.
+              -ez → rapidez, etc.
+            -->
+            <pattern>
+                <marker>
+                    <token inflected='yes'>dar</token>
+                </marker>
+                <and>
+                    <token regexp='yes'>.*de|.*ez
```

Comment:
```suggestion
                    <token regexp='yes'>.+de|.+ez
```

Reply:
Preferring `.+de` (e.g. `dar de comer a` -> *`conferir de comer a`.

Reply:
Also, is there any particular reason for these two suffixes, and not others?
* -mento;
* -idão;
* -itude.

Reply:
Yes, because I was stressed and couldn't think of all the hypotheses.

Feel free to make any changes necessary to have the best rule possible.

❤️ 😋 😛

Reply:
Let's leave it like this for now and see how it performs, then. Later on we can add some suffix entities to the XMLs and experiment. FWIW this is a really good rule.